ui/lib/core/addon/components/radio-select-ttl-or-string.hbs

```diff
@@ -0,0 +1,52 @@
+<div class="column is-narrow is-flex-center has-text-grey has-right-margin-s has-top-margin-negative-s">
```

Comment:
New component that I will use several times in PKI but unsure if it will be used in many other future cases. 

https://user-images.githubusercontent.com/6618863/192022959-34d87831-bcaa-4563-964e-7c2b9148934d.mov